gh-pages/index.md

```diff
@@ -0,0 +1,33 @@
+# What is Baseline?
+
+Web Platform Baseline gives you clear information about which web platform features are ready to use in your projects today. When reading an article, or choosing a library for your project, if the features used are all part of Baseline, you can trust the level of browser compatibility. By aligning with Baseline, there should be no surprises when testing your site.
+
+## How do features become part of Baseline?
+
+Features that are in Baseline are interoperable across a core browser set:
+
+- Chrome (desktop and Android)
+- Edge
```

Comment:
Edge also exists on mobile, so it's worth adding `(desktop)` on this line.

Reply:
fixed